src/entities/User.ts

```diff
@@ -32,6 +37,25 @@ class User {
   })
   googleID: string;
 
+  /** User's graduation year */
+  @Column({
+    type: 'varchar',
+  })
+  graduationYear: string;
+
+  /** User's hometown year */
+  @Column({
+    type: 'varchar',
+  })
+  hometown: string;
```

Comment:
what is a hometown year lool

Reply:
I too have no idea what a hometown year is lol. Am I missing something, where is the issue?

Reply:
On line 46, right above what github displays as a preview, it says `/** User's hometown year */`.